src/modules/store.js

```diff
@@ -0,0 +1,7 @@
+const getTasks = (tasks) => tasks.push(...JSON.parse(localStorage.getItem('tasks')));
```

Comment:
- [ ] At this point in the project, you are expected to have hardcoded tasks. Your tasks should not be loaded from the local storage as you have done. Kindly refactor your code to do that.

Reply:
Hey reviewer, in the showTask.js, I have hard-coded/ created the tasks.